dlna/dms/dms.go

```diff
@@ -813,6 +813,11 @@ func (server *Server) initMux(mux *http.ServeMux) {
 	mux.HandleFunc("/debug/pprof/", pprof.Index)
 	for i, di := range server.Icons {
 		mux.HandleFunc(fmt.Sprintf("%s/%d", deviceIconPath, i), func(w http.ResponseWriter, r *http.Request) {
+                        w.Header().Set("Content-Type", di.Mimetype)
+                        ext, _ := mime.ExtensionsByType(di.Mimetype)
+                        if ext != nil && len(ext) > 0 {
+                                w.Header().Set("Ext", strings.TrimPrefix(ext[0], "."))
```

Comment:
The "Ext" header is only mentioned [here](https://www.w3.org/Protocols/HTTP/ietf-http-ext/Scenarios.html). Looks like it always should be empty, and it is completely unrelated to the file extension.
I set it while debugging issue #40. It just seemed strange to me that the header is empty.
Thank you for removing these lines youself.